### ISSUE TITLE
Broken link to paper in README.md

### ISSUE THREAD

**Issue by katexochen** [jinzhao-dev/jinzhao-disk#38]
Hi, the link in the readme that should point to the paper seems to be broken: https://github.com/jinzhao-dev/jinzhao-disk/blob/master

**Comment by tatetian** [comment]
Yes, it is intentionally left empty as the paper is not ready. We are still improving its quality. Thanks for your interest and stay tuned.

**Comment by katexochen** [comment]
Ah, good to know, thanks! :)
